vms/evm/uptimetracker/uptime_tracker.go

```diff
@@ -57,27 +60,30 @@ func New(
 func (u *UptimeTracker) GetUptime(validationID ids.ID) (
 	time.Duration,
 	time.Time,
-	bool,
 	error,
 ) {
 	u.lock.Lock()
 	defer u.lock.Unlock()
 
 	nodeID, ok := u.state.getNodeID(validationID)
 	if !ok {
-		return 0, time.Time{}, false, nil
+		return 0, time.Time{}, fmt.Errorf(
+			"validationID %s: %w",
+			validationID,
+			errValidationIDNotFound,
+		)
```

Comment:
Small nit - this error format will spit out validation id twice (`validationID <id>: validationID not found` vs `validationID not found: <id>` )
```suggestion
		return 0, time.Time{}, fmt.Errorf(
			"%w: %s",
			errValidationIDNotFound,
			validationID,
		)
```